### PR TITLE
plexguide.md

### PR DESCRIPTION
**Pull request by mrfret**
Please describe the purpose of this Pull Request.
Some minor edits to read better

### CODE REVIEW

**Comment by mrfret** [comment]
minor edits to formatting

File: docs/reference/guides/plexguide.md [hunk 1]
```diff
@@ -32,36 +32,49 @@ AND
 4. If you are not using encryption - Remove gcrypt: and tcrypt: if they exist.
 
 Final should look like:
-'
+```
 [google]
 type = union
 remotes = gcrypt: tcrypt:
```
Comment (chazlarson) [inline docs/reference/guides/plexguide.md:38]: should this be `gdrive: tdrive:`?  The line above says to remove them.

File: docs/reference/guides/plexguide.md [hunk 2]
```diff
@@ -32,36 +32,49 @@ AND
...
 Once this is saved, you need to install the mount service:
 
-'sb install mounts'
+```shell
+sb install mounts
+```
 
 ## Cloudplow changes
 
 If you are using encryption, you definitly need to change this.  If not, unsure but see for yourself:
```
Comment (chazlarson) [inline docs/reference/guides/plexguide.md:49]: definitely

**Review by chazlarson — CHANGES_REQUESTED** [review]

File: docs/reference/guides/plexguide.md (hunk 1 above)
Comment (mrfret) [reply docs/reference/guides/plexguide.md:44]: edits made

File: docs/reference/guides/plexguide.md (hunk 2 above)
Comment (mrfret) [reply docs/reference/guides/plexguide.md]: edits made

**Review by chazlarson — APPROVED** [review]

**Pull request merged**